src/Uno.UI.Runtime.Skia.Gtk/GTK/GtkCoreWindowExtension.cs

```diff
@@ -0,0 +1,230 @@
+							isInContact: false,
+							properties: BuildProperties(args.Event)
+						)
+					)
+				);
+			}
+			catch (Exception e)
+			{
+				this.Log().Error("Failed to raise PointerEntered", e);
+			}
+		}
+
+		private void Window_ButtonReleaseEvent(object o, ButtonReleaseEventArgs args)
+		{
+			try
+			{
+				_ownerEvents.RaisePointerReleased(
+					new PointerEventArgs(
+						new Windows.UI.Input.PointerPoint(
+							frameId: 0,
+							timestamp: args.Event.Time,
+							device: PointerDevice.For(PointerDeviceType.Mouse),
+							pointerId: 0,
+							rawPosition: new Windows.Foundation.Point(args.Event.X, args.Event.Y),
+							position: new Windows.Foundation.Point(args.Event.X, args.Event.Y),
+							isInContact: false,
+							properties: BuildProperties(args.Event)
+						)
+					)
+				);
+			}
+			catch (Exception e)
+			{
+				this.Log().Error("Failed to raise PointerReleased", e);
+			}
+		}
+
+		private void Window_ButtonPressEvent(object o, ButtonPressEventArgs args)
+		{
+			try
+			{
+				_ownerEvents.RaisePointerPressed(
+					new PointerEventArgs(
+						new Windows.UI.Input.PointerPoint(
+							frameId: 0,
+							timestamp: args.Event.Time,
+							device: PointerDevice.For(PointerDeviceType.Mouse),
+							pointerId: 0,
+							rawPosition: new Windows.Foundation.Point(args.Event.X, args.Event.Y),
+							position: new Windows.Foundation.Point(args.Event.X, args.Event.Y),
+							isInContact: false,
+							properties: BuildProperties(args.Event)
+						)
+					)
+				);
+			}
+			catch (Exception e)
+			{
+				this.Log().Error("Failed to raise PointerPressed", e);
+			}
+		}
+
+		private PointerPointProperties BuildProperties(EventButton eventButton)
+			=> new PointerPointProperties
+			{
+				IsLeftButtonPressed = eventButton.Button == 1,
+				IsRightButtonPressed = eventButton.Button == 3,
+			};
+
+		private PointerPointProperties BuildProperties(EventCrossing eventCrossing)
+			=> new PointerPointProperties
+			{
+				IsLeftButtonPressed = (eventCrossing.State & ModifierType.Button1Mask) != 0,
+				IsRightButtonPressed = (eventCrossing.State & ModifierType.Button4Mask) != 0,
+			};
+
+		private void OnMotionEvent(object o, MotionNotifyEventArgs args)
+		{
+			try
+			{
+				switch (args.Event.Type)
+				{
+					case Gdk.EventType.MotionNotify:
+						_ownerEvents.RaisePointerMoved(
+							new PointerEventArgs(
+								new Windows.UI.Input.PointerPoint(
+									frameId: 0,
+									timestamp: args.Event.Time,
+									device: PointerDevice.For(PointerDeviceType.Mouse),
+									pointerId: 0,
+									rawPosition: new Windows.Foundation.Point(args.Event.X, args.Event.Y),
+									position: new Windows.Foundation.Point(args.Event.X, args.Event.Y),
+									isInContact: false,
+									properties: BuildProperties(args.Event)
+								)
+							)
+						);
+						break;
+
+					default:
+						Console.WriteLine($"Unknown event: {args.Event.State}");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)